src/seth/libexec/seth/seth-index

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+### seth-index -- Prints the slot number for the specified mapping type and input data
+### Usage: seth index <fromtype> <totype> <fromvalue> <slot> [<lang>]
+###
+### Prints the slot number for the specified mapping type and input data. For example,
+### the balances mapping in DAI is slot 2, so to find the slot where the balance 
+### is stored for <account>, use
+###     seth-index address uint256 <account> 2
```

Comment:
Can we document the allowed values for `<lang>` here pls

Reply:
Done in ba95b045